lib/pdk/tests/unit.rb

```diff
@@ -21,6 +23,32 @@ def self.invoke(tests, report = nil)
         output = PDK::CLI::Exec.execute(cmd(tests))
         report.write(output) if report
       end
+
+      def self.list
+        PDK::Util::Bundler.ensure_bundle!
+        PDK::Util::Bundler.ensure_binstubs!('rspec-core')
+        command_argv = [File.join(PDK::Util.module_root, 'bin', 'rspec'), '--dry-run', '-fjson']
```

Comment:
My preference is to use the more verbose forms of options in scripts (`--format json` instead of `-fjson`) as it makes the intentions clearer.